trade-finance-manager-api/src/v1/helpers/amendment.helpers.js

```diff
@@ -1,27 +1,201 @@
 const api = require('../api');
 const sendTfmEmail = require('../controllers/send-tfm-email');
-const CONSTANTS = require('../../constants');
-const { amendmentEmailVariables } = require('../emails/amendments/automatic-approval-email-variables');
+const { AMENDMENT_UW_DECISION, AMENDMENT_BANK_DECISION } = require('../../constants/deals');
+const EMAIL_TEMPLATE_IDS = require('../../constants/email-template-ids');
+const { automaticAmendmentEmailVariables } = require('../emails/amendments/automatic-approval-email-variables');
+const { approvedWithConditionsDecision,
```

Comment:
`approvedWithConditionsDecision` should be on a new line

Reply:
done